pkg/reconciler/buildrun/resources/sources.go

```diff
@@ -14,19 +14,42 @@ import (
 
 const defaultSourceName = "default"
 
-// AmendTaskSpecWithSources adds steps, results and volumes for spec.source and spec.sources
+// isLocalCopy appends all "Sources" in a single slice, and if any entry is typed "LocalCopy" it
+// returns first LocalCopy typed BuildSource found, or nil.
+func isLocalCopy(build *buildv1alpha1.Build, buildRun *buildv1alpha1.BuildRun) *buildv1alpha1.BuildSource {
```

Comment:
The name of the function suggests that it returns a boolean. I suggest to chose a name like `findLocalCopyBuildSource`.

Reply:
Indeed, a more complete name makes more sense here. I adopted your suggestion, with a minor modification. Please consider.